### DISCUSSION TITLE
Can we add MuleSoft tags please?

### DISCUSSION THREAD

**Discussion by manikmagar** [BattlesnakeOfficial/feedback#200] [Feature Requests]
I recently created a battlesnake starter for MuleSoft/DataWeave - https://github.com/manikmagar/mule-battlesnake-starter.

It would be nice to have tags - MuleSoft, [DataWeave](https://docs.mulesoft.com/mule-runtime/4.4/dataweave). TIA!

**Comment by bvanvugt** [comment]
Cool, yeah we can definitely do this :) I'll see if we can get it live this week.

**Reply by manikmagar** [reply]
Awesome, Thank you!

**Comment by chris-bsnake** [comment]
Ok, DataWeave and MuleSoft are now added to the system

**Reply by manikmagar** [reply]
Thank you, Chris!
